### ISSUE TITLE
Question about batch size

### ISSUE THREAD

**Issue by lxysl** [LiJunnan1992/DivideMix#53]
Hi, I am impressed and interested in your outstanding work. However, I noticed a question about the batch size while experimenting with the code. When I increase the batch size to 128 or 256, the test accuracy decreases. I am unsure why this algorithm is affected by a larger batch size. Theoretically, the effect of co-guess, co-refine, and mixmatch operators is not related to the batch size. I have two possible reasons for this:
1. the learning rate is ought to increase proportionally with batch size
2. I traced back to the [MixMatch implementation code](https://github.com/YU1ut/MixMatch-pytorch/blob/cc7ef42cffe61288d06eec1428268b384674009a/train.py#L253), and found that there are slight differences with the MixMatch in DivideMix. Between line 253 to line 262, the original code do the interleave operation, but it is not implemented in DivideMix. The author of MixMatch also explain the meaning of this operation in [this issue](https://github.com/google-research/mixmatch/issues/7), which seems to be related to the BatchNorm.

Could you please give some explanations or insights on this question? I will also do some extra experiments on this topic, and will share the results soon. Thank you!
